chimerapy/chimera.py

```diff
@@ -59,3 +66,88 @@ def generate_candidate_mask(m171, m193, m211):
 
     final_mask = mask_171_211 * mask_211_193 * mask_171_193
     return final_mask
+
+def filter_by_area(mask, min_area=100):
+    """
+    Filters the CH mask to keep only coronal holes that have an area greater than or equal to "min_area".
+    """
+    labeled_mask = measure.label(mask)
+    regions = measure.regionprops(labeled_mask)
+
+    filtered_mask = np.zeros_like(mask)
+    for region in regions:
+        if region.area >= min_area:
+            filtered_mask[labeled_mask == region.label] = 1
+
+    return filtered_mask
+
+def get_coronal_holes(ch_mask, map_obj):
+    """
+    Extract coronal hole properties from the binary mask.
+
+    Parameters      
+    ----------
+    ch_mask : `numpy.ndarray
+        The binary mask of coronal holes.
+    map_obj : `sunpy.map.Map
+        The map object from which the mask was generated.
+
+    Returns
+    -------
+    list of dict
+        A list of dictionaries containing properties of each coronal hole.
+    """
+    coronal_holes = []
+    labeled_mask = measure.label(ch_mask)
+
+    solar_radius_arcsec = map_obj.rsun_obs
+    pixel_scale_arcsec = map_obj.scale[0]
+    solar_radius_pixels = (solar_radius_arcsec / pixel_scale_arcsec).value
+    solar_disk_area_pixels = np.pi * (solar_radius_pixels ** 2)
+
+    for region in measure.regionprops(labeled_mask):
+        min_row, min_col, max_row, max_col = region.bbox
+        width_pixels = {
+            "E-W": max_col - min_col,
+            "N-S": max_row - min_row
+        }
```

Comment:
What about transforming all the pixels of the contour to world coordinates first, then transform to heliographic?

I also think want to keep a few of the intermediate values you calculate below e.g min/max of lat and lon as well as the difference or extent